src/main/java/org/jitsi/videobridge/shim/ConferenceShim.java

```diff
@@ -96,38 +87,32 @@ public Conference getConference()
         }
     }
 
-    public ChannelBundleShim getOrCreateChannelBundle(String channelBundleId)
-    {
-        synchronized (channelBundles)
-        {
-            return channelBundles.computeIfAbsent(
-                    channelBundleId,
-                    id -> new ChannelBundleShim(conference, channelBundleId));
-        }
-    }
-
     public Collection<AbstractEndpoint> getEndpoints()
     {
         return conference.getEndpoints();
     }
 
     public void describeChannelBundles(
             ColibriConferenceIQ iq,
-            Set<String> channelBundleIdsToDescribe)
+            Set<String> endpointIds)
     {
-        synchronized (channelBundles)
+        for (AbstractEndpoint endpoint : getEndpoints())
         {
-            channelBundleIdsToDescribe.forEach(bundleId -> {
-                ChannelBundleShim channelBundle = channelBundles.get(bundleId);
-                if (channelBundle != null)
+            String endpointId = endpoint.getID();
+            if (endpointIds.contains(endpointId))
+            {
+                TransportManager transportManager
+                        = ((Endpoint)endpoint).getTransportManager();
```

Comment:
do we need to expose the transport manager here?  can we just ask the endpoint to describe it's transport manager? i.e. `endpoint.describeTransportManager(responseBundleIQ)`

Reply:
I don't know. My thinking was that I want to avoid adding "describe" code to Endpoint. I'll take a look again.

Reply:
Yeah, I did what you suggested